tests/core/test_cloud.py

```diff
@@ -0,0 +1,18 @@
+from kube_hunter.core.events.types.common import NewHostEvent
+
+
+# Testing if it doesn't try to run get_cloud if the cloud type is already set.
+# get_cloud(1.2.3.4) will result with an error
+def test_presetcloud():
+    expcted = "AWS"
+    hostEvent = NewHostEvent(host="1.2.3.4", cloud=expcted)
+    assert expcted == hostEvent.cloud
+
+
+def test_getcloud():
+    AZURE_SERVER = "52.224.188.147" # this is portal.azure.com DNS record
```

Comment:
I think it will be better to use a mock answer (using requests_mock) and not rely on this data to be persistent.